crates/ruff/src/rules/ruff/rules/collection_literal_concatenation.rs

```diff
@@ -48,86 +48,128 @@ fn make_splat_elts(
     new_elts
 }
 
-#[derive(Debug, Copy, Clone)]
+#[derive(Debug, Copy, Clone, PartialEq)]
```

Comment:
```suggestion
#[derive(Debug, Copy, Clone, PartialEq, Eq)]
```

Adding `Eq` here should be fine.

Reply:
@MichaReiser Thank you for the comment.

Adding `Eq` does not cause any issue indeed. I'm a bit new to Rust, is it good practice to add `Eq` along with `PartialEq` wherever possible ?